### DISCUSSION TITLE
Feature: terminal

### DISCUSSION THREAD

**Discussion by TheRealMrKam** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#15] [Ideas]
Hello sir, is it possible to add an option to the EnhancedRB that makes you able to use the terminal by writting in the chat when you are close to the RB please.
I was using this mod before " https://thunderstore.io/c/lethal-company/p/Sheol/RadarTerminal/ "   but it's not working properly in v49.

**Comment by MrHydralisk** [comment]
Saw that RT mod before, but didn't really try it out. It not really open source mod or at least it doesn't have github page from what I saw. And it wouldn't be really nice to copy paste his work, without permission. At the moment I didn't really had plans for ERB additions and I currently a bit busy either way. So even if I give it a try it wouldn't be soon.

Alternatively since RT mod was utilizing LCRemoteTerminal mod code, then it prob would be easier to LCRT dev to implement it as addition to current options. Like an option in config of LCRT for chat terminal working only in proximity to Ship or Radar Booster.

**Reply by TheRealMrKam** [reply]
Thanks for your response, i wanted to ask the radar terminal dev but there is no github to ask him, and because i also use your mod i thought it would be a nice addition !

The LCRemoteTerminal dev also have no github sadly :/

Both mods are also old so idk if they will be maintained.

**Reply by MrHydralisk** [reply]
LCRT seem to be having it version updated after v49 of Lethal Company. I would suggest trying to check modding wiki discord from Thunderstore. It possible that LCRT have thread in it or maybe can search for LCRT related msgs.

**Reply by TheRealMrKam** [reply]
Ok, thank you, i'll try !

edit: No thread sadly, but i found him on discord and asked him on mp, thanks again !

**Reply by TheRealMrKam** [reply]
Well, he did not respond :/

**Reply by MrHydralisk** [reply]
Still better progress, than none. Maybe he respond, but not on daily basis.
